src/workflow-step-template-schema.json

```diff
@@ -3541,6 +3541,12 @@
             },
             "changeReason": {
               "$ref": "#/definitions/workflow-template/change-reason"
+            },
+            "customOptions": {
```

Comment:
how about using existing `ui:options` or `ui:widget`?

Reply:
My thinking was that being able to toggle the ability to enter arbitrary values is not an UI setting, it changes the functionality of the field more in the way of `changeReason`, `prepare` etc imo

Reply:
all of those example you gave have subsequent effect and logic post input..

while this one only changed the input field..

Reply:
Hmm, okay.

I guess this is more for the other PR, but the input still looks identical, just the `Add: "foo"` option no longer shows up. Was I supposed to change the input in any other way?

Reply:
No, why do we need to change it? :confused:

Reply:
So we do want it to look the same regardless of this setting, but we also want this to be a UI setting?